app/views/challenges/show.erb

```diff
@@ -1,90 +1,73 @@
-<% challenge_id(@challenge) %>
+<% challenge_id(@show_challenge) %>
```

Comment:
Any reason why we can't keep existing @challenge? It's not clear what `show_` adds in terms of readability.

Reply:
It was to show to other devs that it is not a 'Challenge model' but 'service/show_challenge' to avoid confusion.

But I agree with you 'ShowChallenge' was a bad name, and so this @show_challenge